<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PresentationParametersControlWidget</class>
 <widget class="QWidget" name="PresentationParametersControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>227</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PresentationParameters</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="8" column="0" colspan="2">
    <widget class="QCheckBox" name="produce3DCheckBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Construct 3D output ON/OFF</string>
     </property>
     <property name="text">
      <string>Produce 3D</string>
     </property>
     <property name="tags" stdset="0">
      <string>advanced</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QCheckBox" name="autoUpdateHistogramCheckBox">
     <property name="text">
      <string>Auto Update Histogram</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QCheckBox" name="produce6DCheckBox">
     <property name="text">
      <string>Produce 6D</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QCheckBox" name="showAreaOfInterestCheckBox">
     <property name="text">
      <string>Area of Interest</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QCheckBox" name="showClustersCheckBox">
     <property name="toolTip">
      <string>Show cluster information</string>
     </property>
     <property name="text">
      <string>Clusters</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QComboBox" name="outputComboBox">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>right frame</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/new/prefix1/right-cam.png</normalon>
       </iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>left frame</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/new/prefix1/left-cam.png</normalon>
       </iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>side-by-side stereo</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/new/prefix1/2cam-active.png</normalon>
       </iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Anaglyph R/G</string>
      </property>
      <property name="icon">
       <iconset resource="../../../resources/main.qrc">
        <normaloff>:/new/prefix1/3d_glasses_rg.png</normaloff>:/new/prefix1/3d_glasses_rg.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Anaglyph R/C</string>
      </property>
      <property name="icon">
       <iconset resource="../../../resources/main.qrc">
        <normaloff>:/new/prefix1/3d_glasses_rc.png</normaloff>:/new/prefix1/3d_glasses_rc.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Blend</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/new/prefix1/2cam-blend-b.png</normalon>
       </iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>All</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QCheckBox" name="dump3DCheckBox">
     <property name="text">
      <string>Dump 3D buffer</string>
     </property>
     <property name="tags" stdset="0">
      <string>advanced</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QComboBox" name="flowComboBox">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Stereo presentaton mode</string>
     </property>
     <item>
      <property name="text">
       <string>don't show flow</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>fast colored dots</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>show lines only</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>show colored dots</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>show colored lines</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>show heat coloring</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QCheckBox" name="showHistogramCheckBox">
     <property name="toolTip">
      <string>show cluster Histogram</string>
     </property>
     <property name="text">
      <string>Histogram</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="5">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QComboBox" name="stereoComboBox">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Stereo presentaton mode</string>
     </property>
     <item>
      <property name="text">
       <string>don't show stereo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>fast large dots</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>fast dots stereo </string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>show lines stereo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>show all stereo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>show dots stereo</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QCheckBox" name="dumpSceneJSONCheckBox">
     <property name="text">
      <string>Dump Scene JSON</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../../resources/main.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>stereoComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>PresentationParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>flowComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>PresentationParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>outputComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>PresentationParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>557</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>produce3DCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>PresentationParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>667</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showClustersCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>PresentationParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>487</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showHistogramCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>PresentationParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showAreaOfInterestCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>PresentationParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeParameters()</slot>
 </slots>
</ui>
